Core.Auditing/Core.Auditing.csproj

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
How is it that we did not have this before and it was working?

Reply:
Transitive dep most likely :)

Reply:
We actually don't need this here :)